### ISSUE TITLE
Cannot persuade version 0.1.4 to run proc_structural

### ISSUE THREAD

**Issue by araikes** [MICA-MNI/micapipe#58]
*   **What is the current behavior?**  
I pulled the current version of micapipe (v. 0.1.4) today in a Singularity image for use on my HPC. To check everything running, I was going to run single participant in very stepwise progression to make sure I understood the outputs at each step.  My intention is to reuse my already run Freesurfer 7.3.1 data but is somewhat optional. This is my call:

```
singularity run --cleanenv \
-B /xdisk/adamraikes/cornell/nifti:/nifti:ro \
-B /xdisk/adamraikes/cornell/derivatives/freesurfer-7.3.1:/fs_input \
-B /xdisk/adamraikes/cornell/derivatives/micapipe:/output \
-B /tmp:/tmp \
-B /groups/adamraikes/license.txt:/opt/freesurfer-6.0.0/license.txt \
/groups/adamraikes/singularity_images/micapipe-v0.1.4.simg 
-bids /nifti \
-out /output \
-sub 087 \
-threads 8 \
-proc_structural
```

This is the output when I do so:

```
Some packages in this Docker container are non-free
If you are considering commercial use of this container, please consult the relevant license:
https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Licence
-------- freesurfer-Linux-centos6_x86_64-stable-pub-v6.0.0-2beb96c --------
Setting up environment for FreeSurfer/FS-FAST (and FSL)
FREESURFER_HOME   /opt/freesurfer-6.0.0
FSFAST_HOME       /opt/freesurfer-6.0.0/fsfast
FSF_OUTPUT_FORMAT nii.gz
SUBJECTS_DIR      /opt/freesurfer-6.0.0/subjects
MNI_DIR           /opt/freesurfer-6.0.0/mni
FSL_DIR           /opt/fsl-6.0.3

[ WARNING ]..... DEPRECATION: mica-pipe is deprecated and will be removed from upcoming versions (use micapipe instead) 


micapipe -bids /nifti -out /output -sub 087 -threads 8 -proc_structural


-------------------------------------------------------------
	MICA pipeline - (Version v0.1.4 'Roadrunner') 
		Subject: 087 Session: SINGLE
-------------------------------------------------------------

[ WARNING ]..... fix was not found, ICA-FIX will be skipped 

[ INFO ]..... micapipe will use 8 threads for multicore processing 

[ INFO ]..... Subject 087 directory doesn't exist 

adamraikes @ r4u11n1.puma.hpc.arizona.edu Mon Nov 21 12:30:26 MST 2022:
COMMAND -->   mkdir -p /output/micapipe/sub-087/logs /output/micapipe/sub-087/xfm /output/micapipe/sub-087/dwi /output/micapipe/sub-087/func /output/micapipe/sub-087/anat/first /output/micapipe/sub-087/anat/volumetric /output/micapipe/sub-087/anat/surfaces/conte69 /output/micapipe/sub-087/QC/png  

adamraikes @ r4u11n1.puma.hpc.arizona.edu Mon Nov 21 12:30:26 MST 2022:
COMMAND -->   ln -s /opt/freesurfer-6.0.0/subjects/fsaverage5/ /output/freesurfer  

adamraikes @ r4u11n1.puma.hpc.arizona.edu Mon Nov 21 12:30:26 MST 2022:
COMMAND -->   mkdir -p /output/freesurfer/conte69/surf  

[ INFO ]..... sub-087 directory structure:
 
/output/micapipe/sub-087
\u251c\u2500\u2500 anat
\u2502   \u251c\u2500\u2500 first
\u2502   \u251c\u2500\u2500 surfaces
\u2502   \u2502   \u2514\u2500\u2500 conte69
\u2502   \u2514\u2500\u2500 volumetric
\u251c\u2500\u2500 dwi
\u251c\u2500\u2500 func
\u251c\u2500\u2500 logs
\u251c\u2500\u2500 QC
\u2502   \u2514\u2500\u2500 png
\u2514\u2500\u2500 xfm

11 directories
/opt/micapipe/micapipe: line 666: /opt/micapipe/functions/01_proc-struc_volumetric.sh /nifti 087 /output/micapipe SINGLE FALSE 8 /tmp DEFAULT: No such file or directory
/opt/micapipe/micapipe: line 949: [: too many arguments


-------------------------------------------------------------
	GLOBAL micapipe running time with container-micapipe v0.1.4 processing: 0.000 minutes 
-------------------------------------------------------------
```

It generates the relevant file tree as expected but fails after creating the folders. The same command works for my v0.1.2 singularity image.

**Comment by rcruces** [comment]
Yes, we found that bug on the release and  it is fixed, but we are currently fixing building the new container that will replace that version. You could build the container from the Docker file provide in master. That branch is already working. We are sorry for the inconvenience.

**Comment by araikes** [comment]
Ah, sounds good. Glad it was a known issue and not a new problem.

Thanks

**Comment by araikes** [comment]
Apologies @rcruces or @PeerHerholz, one other question...

In `master` the Dockerfile still has FSL at v. 6.0.0 vs. 6.0.3 as is up on DockerHub. Are the only updates needed there to update to change relevant paths (both from FSL and for the install path) to 6.0.3?

**Comment by araikes** [comment]
@rcruces and @PeerHerholz,

Curious if 0.1.5 is going to get a build on Dockerhub? I'd build it myself but the Dockerfile in `master` hasn't been updated in 2 years, so I'm assuming it to be out of date.

**Comment by rcruces** [comment]
We just updated some features on the new release, and we were having some difficulties while building the docker. But we will updated and upload a new container possibly by next week. We'll keep you post.

**Comment by rcruces** [comment]
This feature was fixed on the `v0.1.5` and the container usage was deeply test for our latest release `v0.2.0` please have a look at it.
https://github.com/MICA-MNI/micapipe/releases/tag/v0.2.0
